packages/date-picker/src/vaadin-date-picker-mixin.d.ts

```diff
@@ -8,7 +8,7 @@ import type { DisabledMixinClass } from '@vaadin/component-base/src/disabled-mix
 import type { FocusMixinClass } from '@vaadin/component-base/src/focus-mixin.js';
 import type { KeyboardMixinClass } from '@vaadin/component-base/src/keyboard-mixin.js';
 import type { DelegateFocusMixinClass } from '@vaadin/field-base/src/delegate-focus-mixin.js';
-import type { InputMixinClass } from '@vaadin/field-base/src/input-mixin.js';
+import type { InputConstraintsMixinClass } from '@vaadin/field-base/src/input-constraints-mixin.js';
```

Comment:
Please keep the `InputMixinClass` as well, it's needed for `vaadin-date-picker-light`. Currently, this fails:

```ts
assertType<InputMixinClass>(datePickerLight);
```

The `vaadin-date-picker` doesn't have this problem as it applies `InputControlMixin` that wraps `InputMixin`.

Reply:
`InputMixin` already comes with `InputConstraintsMixin`. Should we still keep it given that?

Reply:
Yes, in case of mixins we have to do it. The `DatePickerMixinClass` applies `InputConstraintsMixinClass` that does not extend `InputMixinClass`, so the `inputElement` would no longer be available in typings.

Reply:
Ah, you are right. Fixed.